resources/sdk/clisdkclient/templates/restclient.mustache

```diff
@@ -243,6 +283,93 @@ func authorize(c config.Configuration) (models.OAuthTokenData, error) {
 	return *oAuthTokenResponse, nil
 }
 
+// Starts the local server at the redirect URI
+func startLocalServerFunc(c config.Configuration, authChannel chan models.OAuthToken) {
+	address := strings.Replace(c.RedirectURI(), "http://localhost", "", -1)
+	port := address[0:5]
+	path := strings.Replace(address, port, "", -1)
+	if path == "" {
+		path = "/"
+	}
+
+	var oAuthToken models.OAuthToken
+	implicitWebpage := models.ImplicitWebpage{}
+	// Set initial web page as one containing the javascript to perform implicit login
+	initialPage := implicitWebpage.GetImplicitWebpage(c.ClientID(), c.Environment(), c.RedirectURI())
+	activePage := initialPage
+
+	http.HandleFunc(path, func(w http.ResponseWriter, r *http.Request) {
+		_, err := fmt.Fprintf(w, activePage)
+		if err != nil {
+			logger.Fatal(err)
+		}
+
+		// If no longer displaying initial page -
+		// Give time for initialPage to refresh on the browser thereby printing the response page,
+		// Then close channel & server
+		if activePage != initialPage {
+			time.Sleep(4 * time.Second)
+			authChannel <- oAuthToken
+			close(authChannel)
+			return
+		}
+
+		if strings.Contains(r.URL.String(), "/error/") || strings.Contains(r.URL.String(), "/access_token/") {
+			oAuthToken = getImplicitResponseDataFromURL(r.URL.String())
+			activePage = implicitWebpage.GetResponsePage(oAuthToken.Error)
+		}
+	})
+
+	if c.SecureLoginEnabled() {
```

Comment:
There's some warning logging coming from the HTTP package when we use HTTPS. Call `log.SetOutput(ioutil.Discard)` before `http.ListenAndServe` to ignore it

Reply:
I think you unresolved this by accident